src/main/java/com/miliancode/model/User.java

```diff
@@ -30,6 +40,44 @@ public class User {
     @Column(nullable = false)
     private String lastName;
     private String shippingAddress;
-    @Column(nullable = false)
+    @Column(name = "is_deleted", nullable = false)
     private boolean isDeleted = false;
+    @ToString.Exclude
+    @ManyToMany(fetch = FetchType.EAGER)

```

Comment:
```suggestion
    @ManyToMany
```

Reply:
Without it, I receiving LazyInitializationException when trying to do any operation with books: failed to lazily initialize a collection of role: com.miliancode.model.User.roles: could not initialize proxy - no Session

Reply:
Eeager is a bad approach, we have some options to resolve such exception, like 
`@EntityGraph(attributePaths = "roles")` under the method in UserRepo 
or
you can use join fetch in query (not preferable)